book/src/fork-selection.md

```diff
@@ -0,0 +1,147 @@
+# Fork Selection
+This chapter defines Solana's *Nakomoto Fork Selection* algorithm based on time
+locks. It satisfies the following properties:
+
+
+* A voter can eventually recover from voting on a fork that doesn't become the
+  fork with the desired finality.
```

Comment:
I think it would help to move the definition of finality above this. This term "desired finality" still doesn't make sense to me.

Reply:
how about *network confirmation threshold*?